core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.component.spec.ts

```diff
@@ -144,9 +151,41 @@ describe('Edit Thumbnail Modal Component', () => {
     component.onFileChanged(file);
     expect(component.invalidImageWarningIsShown).toBe(false);
     expect(component.invalidFilenameWarningIsShown).toBe(false);
-    expect(component.tags).toEqual(['script']);
-    expect(component.attrs).toEqual([]);
-    expect(resetSpy).toHaveBeenCalled();
+  });
+
+  it('should remove invalid tags and attributes', ()=>{
+    spyOn(component, 'isUploadedImageSvg').and.returnValue(true);
+    spyOn(component, 'isValidFilename').and.returnValue(true);
+    spyOn(
+      svgSanitizerService, 'getInvalidSvgTagsAndAttrs'
+    ).and.callFake(() => {
+      return { tags: ['circel'], attrs: ['data-name'] };
+    });
+    const svgString = (
```

Comment:
It seems like this svg string can be arbitrary and it won't affect the result? I think it might be better, for the svgSanitizerService above, to use a callThrough with verification that the result is correct, rather than a spy that mocks the svgSanitizerService.

Reply:
Done!